applications/endoscopy_tool_tracking_distributed/README.md

```diff
@@ -0,0 +1,37 @@
+## Distributed Endoscopy Tool Tracking Application
+
+Digital endoscopy is a key technology for medical screenings and minimally invasive surgeries. Using real-time AI workflows to process and analyze the video signal produced by the endoscopic camera, this technology helps medical professionals with anomaly detection and measurements, image enhancements, alerts, and analytics.
+
+
+![](docs/app_endoscopy.png)<br>
+Fig. 1 Endoscopy (laparoscopy) image from a cholecystectomy (gallbladder removal surgery) showing AI-powered frame-by-frame tool identification and tracking. Image courtesy of Research Group Camma, IHU Strasbourg and the University of Strasbourg ([NGC Resource](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_endoscopy_sample_data))
+
+
+
+The Distributed Endoscopy tool tracking application provides an example of how an endoscopy data stream can be captured and processed using the C++ or Python APIs on multiple hardware platforms.
```

Comment:
Could you please add another sentence or two to describe 1) what is a Holoscan Fragment and where can the user find more information on it, and 2) is there an advantage to running the application in a distributed manner like this?

```suggestion
The Distributed Endoscopy Tool Tracking application provides an example of how an endoscopy data stream can be captured and processed using the C++ or Python APIs on multiple hardware platforms.
```